statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -143,16 +140,16 @@ func (wr *WidgetsRender) prepareInstanceInfo() {
 		consensusInfo = fmt.Sprintf("Consensus accepted / signed blocks: %d / %d", countConsensusAcceptedBlocks, countConsensus)
 	}
 
-	rows[4] = []string{consensusInfo}
+	rows[3] = []string{consensusInfo}
 
 	countLeader := wr.presenter.GetCountLeader()
 	countAcceptedBlocks := wr.presenter.GetCountAcceptedBlocks()
-	rows[5] = []string{fmt.Sprintf("Blocks accepted / blocks proposed : %d / %d", countAcceptedBlocks, countLeader)}
+	rows[4] = []string{fmt.Sprintf("Blocks accepted / blocks proposed : %d / %d", countAcceptedBlocks, countLeader)}
```

Comment:
Maybe check all strings here as to not have [_white_space_][punctuation] such as ` : %d / %d` pattern in strings

Reply:
done.